src/main/resources/assets/botania/shader/halo.frag

```diff
@@ -11,7 +11,7 @@ void main() {
    
    for(int i = -4 ; i < 4; i++)
         for(int j = -3; j < 3; j++)
-            sum += texture2D(bgl_RenderedTexture, texcoord + vec2(j, i) * 0.004) * 0.25;
+            sum += texture2D(bgl_RenderedTexture, texcoord + vec2(0, 0) * 0.004) * 0.25;
```

Comment:
can't this just be reduced to `texcoord` now?

Reply:
Yes that's a good point